website/aggregated_questions.qmd

```diff
@@ -0,0 +1,251 @@
+# (Preliminary) Questions
+
+_The following section contains questions for consideration (pruning, subsectioning, and or rephrasing) in GET. Included are questions where some work has already been expended (thus, the author is less inclined to prune these questions), where little work has been expended (full critique welcome here), and questions that already exist on Metaculus. Ratings of importance for the questions is not included here yet but will be once they have been established. Information in brackets are the author's notes._
+
+## Ethical, Legal, & Societal Implications
+
+Will an international treaty [treaty required] regulating human genetic engineering be signed by 20XX? (note: conditional on legal genetic engineering)
+
+Will there cease to be a global moratorium on human germline genome editing by 20XX?
```

Comment:
There is no international law about this now, so "global moratorium" should be clarified